packages/components/warning-callout/template.njk

```diff
@@ -3,13 +3,13 @@
 {%- if params.classes %} {{ params.classes }}{% endif %}"
 {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
   <h{{ headingLevel }} class="nhsuk-warning-callout__label">
-    {%- if params.heading != "Important" %}
+    {%- if "Important" or "important" in params.heading %}
```

Comment:
I couldn't get this to work when having "Important" or "important" in the heading.
I had to use this:
`{%- if "Important" in params.heading or "important" in params.heading %}`
Could you confirm this works for you too?